src/crs/__init__.py

```diff
@@ -164,7 +164,14 @@
         ),
         max_bbox=None,
         no_area_bbox_padding=1,
-        world_bounds=dict(),
+        world_bounds={
+            EPSG_4326: (
```

Comment:
Add comment: Extent is just a copy of EPSG:3857 world bounds that works for DB ST_Transform, larger extent could also work, we did not try.

Reply:
Or enlarge world_bounds of EPSG_3034 and EPSG_3857 to max tested value that works for ST_Transform